libcudacxx/include/cuda/std/__type_traits/type_list.h

```diff
@@ -326,16 +329,22 @@ using __type_push_front = __type_call1<_List, __type_bind_front_quote<__type_lis
 namespace __detail
 {
 template <template <class...> class _Fn, class... _Ts>
-_LIBCUDACXX_HIDE_FROM_ABI auto __as_type_list_fn(_Fn<_Ts...>*) -> __type_list<_Ts...>;
+_LIBCUDACXX_HIDE_FROM_ABI auto __as_type_list_fn(__undefined<_Fn<_Ts...>>*) //
```

Comment:
Why is there a need for the manual formatting?

Reply:
if i don't, then clang-format will format one overload one way, and another (with a longer return type) a different way, making it harder for me to read. i like being able to scan down and find the arguments and the return types in generally the same place for each overload.